### DISCUSSION TITLE
Css language sever

### DISCUSSION THREAD

**Discussion by N4tus** [helix-editor/helix#4670] [Q&A] [answered]
The language server for css is defined as `vscode-css-language-server`. After finding `vscode-css-languageserver` (https://github.com/microsoft/vscode-css-languageservice) I tried to install it, but it does not seem to expose something to execute as language server.  

I don't know if this is even the intended server, so I want to ask how it should be installed properly.

**Comment by kirawi** [comment, marked as answer]
You can check the wiki: https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#cssls
